.changeset/funny-glasses-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+"@svelte-add/adders": patch
+---
+
+fix: Replaced `@tailwind` directives for `@import`
```

Comment:
I don't know if you guys have any convention for this, but in SvelteKit we always start with a lowercase letter. Just a matter of opinion and pretty subjective though
```suggestion
fix: replaced `@tailwind` directives for `@import`
```